homework1/S5.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Scanner;
+
+public class S5 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String s = sc.nextLine();
+
+        boolean flag = Palindrome(s);
```

Comment:
use more meaningful variable names like isPalindrome from next time

Reply:
Made the necessary changes regarding naming conventions